x-pack/plugins/reporting/server/lib/level_logger.ts

```diff
@@ -10,7 +10,14 @@ const trimStr = (toTrim: string) => {
   return typeof toTrim === 'string' ? toTrim.trim() : toTrim;
 };
 
-export class LevelLogger {
+export interface ShareableLogger {
+  debug: (msg: string, tags?: string[]) => void;
+  info: (msg: string, tags?: string[]) => void;
+  warning: (msg: string, tags?: string[]) => void;
+  error: (msg: string | Error, tags?: string[]) => void;
```

Comment:
The error method on the tooling log is a little less flexible than this, it only accepts a single error object which is prints.